tutorials/introductory/sample_plots.py

```diff
@@ -168,10 +202,10 @@
 
 .. _screenshots_table_demo:
 
-Table demo
+Tables
 ==========
```

Comment:
Same as before, but would it be possible to match the length of the '====' line with the line beyond?